extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -535,6 +537,85 @@ def from_dict(self, model):
         self._c_lw = copy.deepcopy(model['_c_lw'])
         self._c_l = copy.deepcopy(model['_c_l'])
 
+    def _make_json_serializable(self, model):
+        """Makes the StringClassifier dict JSON serializable by
+        converting numpy data types to Python data types.
+
+        Args:
+            model: A dict representing a StringClassifier.
+
+        Returns:
+            model: A dict representing a StringClassifier
+        """
+
+        conversion_list_of_list_properties = [
+            '_b_dl',
+            '_c_dl',
+            '_c_lw'
+        ]
+
+        conversion_list_properties = [
+            '_c_l'
+        ]
+
+        conversion_properties = [
+            '_l_dp'
+        ]
+
+        for classifier_property in conversion_list_of_list_properties:
```

Comment:
So, stuff like this doesn't work? https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.ndarray.tolist.html